src/transpiling/mod.rs

```diff
@@ -1389,7 +1389,7 @@ const a = _jsx(Foo, {
     assert_eq!(
       emit_result.source_map.as_deref(),
       Some(
-        r#"{"version":3,"sources":["https://deno.land/x/mod.tsx"],"sourcesContent":["{ const foo = \"bar\"; };"],"names":[],"mappings":"AAAA;EAAE,MAAM,MAAM;AAAO"}"#
+        r#"{"version":3,"sources":["https://deno.land/x/mod.tsx"],"sourcesContent":["{ const foo = \"bar\"; };"],"names":[],"rangeMappings":";;","mappings":"AAAA;EAAE,MAAM,MAAM;AAAO"}"#
```

Comment:
What's that for?

Reply:
Seems to be https://github.com/tc39/source-map/blob/main/proposals/range-mappings.md